wordpress-mobile/WordPress-iOS
# Basic Background on Signup Header/Footer

Adds `.basicBackground` to signup's header and footer cells.

Fixes #16073

To test:
1. Sign up for new WordPress account.
2. Note that header and footer cells on confirmation page have background that match the rest of the page.
3. Switch to dark mode.
4. Confirm that same thing happens.

### Screenshots
| Before | After - Light | After - Dark
| - | - | - |
| <img src="https://user-images.githubusercontent.com/13263478/111363606-4190f980-865e-11eb-8958-d2e7649cf2ea.png" width="100%"/> | <img src="https://user-images.githubusercontent.com/13263478/111363575-3a69eb80-865e-11eb-8a22-d75eb40a3d06.png" width="100%"/> | <img src="https://user-images.githubusercontent.com/13263478/111363583-3ccc4580-865e-11eb-8e34-60e3577fcb7d.png" width="100%"/> |

PR submission checklist:
